networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -6,4 +6,23 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+As a cluster administrator, you can configure an additional network for a virtual routing and forwarding (VRF) domain by using the CNI VRF plugin. The virtual network created by this plugin is associated with a physical interface that you specify. 
+
+Using a secondary network with a VRF instance has the following advantages:
+
+Workload isolation:: Isolate workload traffic by configuring the additional network's VRF instance. 
+Improved security:: Enable improved security through isolated network paths in the VRF domain.
+Multi-tenancy support:: Support multi-tenancy through network segmentation with a unique routing table in the VRF domain for each tenant.
```

Comment:
ISG says not to use a hyphen here, but I imagine that's a common word in this content area. Would leave it alone for consistency, but consider updating the overall content to use "multitenancy" as a separate effort.